notebooks/contributing/formatting.md

```diff
@@ -0,0 +1,289 @@
+Basic functionality within SimPEG is not discussed in detail, as we assume
+the user is already familiar.
+:::
+```
+
+```
+:::{admonition} Advanced notebook
+:class: danger
+This tutorial focusses on advanced functionality within SimPEG. Basic and
+intermediate level functionality are not discussed in detail, as we assume
+the user is already an experienced SimPEG user.
+:::
+```
+
+### Cell 3: Computational Requirements
+
+Here, we specify the computational requirements to run the notebook. Whereas some notebooks can be run easily on laptop computers, some notebook may need to be exported as Python files and run using cluster computing. The author is required to choose from one of the following [MyST admonitions](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html), which is rendered when the webpage is built.
+
+```
+:::{admonition} Light-weight notebook
+:class: hint
+This tutorial requires minimal computational resources and can be executed
+quickly in the background while other computer processes are running.
+:::
+```
+
+```
+:::{admonition} Medium-weight notebook
+:class: caution
+Requires moderate computational resources. Run-times may exceed several
+minutes and require up to 8 GB of available RAM.
+:::
+```
+
+```
+:::{admonition} Computationally intensive notebook
+:class: danger
+The computational resources required to execute this notebook exceed those
+provided by standard laptop computers. To execute the notebook, please deploy
+to a cluster computing environment.
+:::
+```
+
+### Cell 4: Keywords, Summary and Learning Objectives
+
+In this cell, the contributor must provide a set of relevant keywords. E.g.
+
+```
+**Keywords:** gravity inversion, sparse-norm inversion, integral formulation, tree mesh.
+```
+
+Next, the contributor must provide a summary of the tutorial. The summary should be roughly one paragraph. If your tutorial focusses on particular functionality within SimPEG, please link to the API documentation. Also, link to tutorials you feel the reader should have already worked through. E.g. 
+
+```
+**Summary:** Here we invert gravity anomaly data to recover a density contrast model.
+We demonstrate two approaches for recovering a density contrast model:
+
+1. Weighted least-squares inversion for a tensor mesh
+2. Iteratively re-weighted least-squares (IRLS) inversion for a tree mesh
+
+The *weighted least-squares* approach is a great introduction to geophysical inversion
+with SimPEG. One drawback however, is that it recovers smooth structures which may not
+be representative of the true model. To recover sparse and/or blocky structures, we
+also demonstrate the *iteratively re-weighted least-squares* approach. Because this
+tutorial focusses primarily on inversion-related functionality, we urge the reader to
+become familiar with functionality explained in the [3D Forward Simulation of Gravity
+Anomaly Data](fwd_gravity_anomaly_3d.ipynb) tutorial before working through this one.
+```
```

Comment:
```suggestion
```markdown
**Summary:** Here we invert gravity anomaly data to recover a density contrast model.
We demonstrate two approaches for recovering a density contrast model:

1. Weighted least-squares inversion for a tensor mesh
2. Iteratively re-weighted least-squares (IRLS) inversion for a tree mesh

The *weighted least-squares* approach is a great introduction to geophysical inversion
with SimPEG. One drawback however, is that it recovers smooth structures which may not
be representative of the true model. To recover sparse and/or blocky structures, we
also demonstrate the *iteratively re-weighted least-squares* approach. Because this
tutorial focusses primarily on inversion-related functionality, we urge the reader to
become familiar with functionality explained in the [3D Forward Simulation of Gravity
Anomaly Data](fwd_gravity_anomaly_3d.ipynb) tutorial before working through this one.
```
```